### PR TITLE
Cleanup warnings

### PR DESCRIPTION
**Pull request by danielSanchezQ**
Just some poldercast warnings are left.

### CODE REVIEW

File: testing/jormungandr-testing-utils/src/testing/node/rest/load.rs [hunk 1]
```diff
@@ -109,6 +109,6 @@ impl RequestGenerator for RestRequestGen {
             }
             _ => unreachable!(),
         }
-        Ok(None)
+        Ok(Vec::new())
```
Comment (danielSanchezQ) [inline testing/jormungandr-testing-utils/src/testing/node/rest/load.rs:112]: @dkijania Is this ok? It feels a bit weird to have this.

File: testing/jormungandr-testing-utils/src/testing/node/rest/raw.rs [hunk 2]
```diff
@@ -187,6 +187,7 @@ impl RawRest {
     }
 
     pub fn send_fragment(&self, fragment: Fragment) -> Result<Response, reqwest::Error> {
+        #[allow(unused_imports)]
         use chain_core::property::Fragment as _;
```
Comment (mzabaluev) [inline testing/jormungandr-testing-utils/src/testing/node/rest/raw.rs:191]: Why is this needed?

Comment (danielSanchezQ) [reply testing/jormungandr-testing-utils/src/testing/node/rest/raw.rs:191]: Looks like for nothing. They were some leftovers and I just automatically added the macro flag. Gonna be removed

**Review by mzabaluev — APPROVED** [review]
